### ISSUE TITLE
Show docset download size

### ISSUE THREAD

**Issue by trollixx** [zealdocs/zeal#353]

**Comment by dufferzafar** [comment]
:+1:

**Comment by trollixx** [comment]
Will be part of #354.

**Comment by Sathors** [comment]
Indeed, seeing the size of the download would be nice, along with the downloading speed.

**Comment by smsrkr** [comment]
I see #354 is closed. But this doesn't seem to be implemented - don't see sizes in the docsets mgmt dialog, not even when downloading.

**Comment by jacksgt** [comment]
Any update on this? I'd also like to see this feature implemented

**Comment by trollixx** [comment]
No updates yet as we are trying to get the next release out.

**Comment by alireza6677** [comment]
This feature is necessary and useful, please consider adding it to the future releases.

**Comment by techtonik** [comment]
Was about to fill the same issue. What is the problem with showing size? It already shows percent that is calculated from it, no?

**Comment by techtonik** [comment]
Here is the code that could be updated with file size data - https://github.com/zealdocs/zeal/blob/769504b020123d81c541c088a7b68e9c53e6d589/src/libs/ui/docsetsdialog.cpp#L238

**Comment by trollixx** [comment]
The idea here is to show the size upfront, not after download has started. Although why not have both features. I think the easiest option would be replacing percentage with the size information.

**Comment by techtonik** [comment]
Although seems easy, C++ and is not my language with its & references. )

**Comment by thetywat** [comment]
![capture1](https://user-images.githubusercontent.com/8014853/48980738-6ebae600-f0fb-11e8-940f-1189ae48b587.PNG)
